src/Components/OATPropertyEditor/PropertyListItemNest.tsx

```diff
@@ -207,6 +207,16 @@ export const PropertyListItemNest = ({
         });
     };
 
+    const showObjectPropertySelector = useMemo(() => {
+        if (item.schema['@type'] === 'object') {
+            if (item.schema.fields.length === 0) {
+                return true;
+            } else if (item.schema.fields.length > 0 && collapsed) {
+                return true;
+            }
```

Comment:
We should be able to collapse this whole block into a return statement...

`return item.schema['@type'] === 'object' && (item.schema.fields.length === 0 || (item.schema.fields.length > 0 && collapsed));` 

Or something like that?